### PR TITLE
Tabard of the Hand of Argus

### PR DESCRIPTION
**Pull request by arithon**
## Changelog:
- Added a tabard for the Hand of Argus. Giving it to Kuros, Boros, and Aesom.

# How to test:
- Start game at any date
- Wait for Draenei to appear (or use console: `event WCDRN.99`)

### CODE REVIEW

File: common/artifact_spawns/wc_artifact_spawns.txt [hunk 1]
```diff
@@ -1126,6 +1126,28 @@ lights_vengeance = {
...
+	}
+
+	weight = {
+		value = 0
+		additive_modifier = {
+			value = 100
+			character = 2107247					# Boros
+			character = 2107248					# Kuros
+			character = 2107250					# Aesom
+		}
```
Comment (zumbak04) [inline common/artifact_spawns/wc_artifact_spawns.txt:1149]: These characters don't exist.

File: common/artifacts/906_wc_artifacts_ceremonial_torso.txt [hunk 2]
```diff
@@ -70,5 +70,28 @@ tabard_of_the_silverhand = {
...
+}
+argus_tabard = {
+	quality = 1
+	flags = { misc uncommon }
+
+	active = {
+		is_adult = yes
+		OR = {
+			religion = naaru
+		}
```
Comment (zumbak04) [inline common/artifacts/906_wc_artifacts_ceremonial_torso.txt:83]:
```suggestion
		religion = naaru
```

File: common/artifacts/906_wc_artifacts_ceremonial_torso.txt [hunk 3]
```diff
@@ -70,5 +70,28 @@ tabard_of_the_silverhand = {
...
+	}
+	allowed_gift = {
+		religion_group = holy_light
```
Comment (zumbak04) [inline common/artifacts/906_wc_artifacts_ceremonial_torso.txt:86]:
```suggestion
		religion_group = light_group
```

File: common/artifact_spawns/wc_artifact_spawns.txt (hunk 1 above)
Comment (arithon) [reply common/artifact_spawns/wc_artifact_spawns.txt:1149]: removed

File: common/artifacts/906_wc_artifacts_ceremonial_torso.txt (hunk 2 above)
Comment (arithon) [reply common/artifacts/906_wc_artifacts_ceremonial_torso.txt:83]: good point, dunno what i thought there

File: common/artifacts/906_wc_artifacts_ceremonial_torso.txt (hunk 3 above)
Comment (arithon) [reply common/artifacts/906_wc_artifacts_ceremonial_torso.txt:86]: fixed

**Pull request merged**